src/private/remoteCamera.ts

```diff
@@ -0,0 +1,359 @@
+    deviceState: DeviceState;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to track the control session status.
+   */
+  export interface SessionStatusChanged {
+    sessionStatus: SessionStatus;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to indicate the current state of the device.
+   */
+  export interface DeviceState {
+    /**
+     * All operation are available to apply.
+     */
+    available: boolean;
+    /**
+     * Either camera doesnt support to get state or It unable to apply command.
+     */
+    error: boolean;
+    /**
+     * Reset max out or already applied. Client Disable Reset.
+     */
+    reset: boolean;
+    /**
+     * ZoomIn maxed out.
+     */
+    zoomIn: boolean;
+    /**
+     * ZoomOut maxed out.
+     */
+    zoomOut: boolean;
+    /**
+     * PanLeft reached max left.
+     */
+    panLeft: boolean;
+    /**
+     * PanRight reached max right.
+     */
+    panRight: boolean;
+    /**
+     * TiltUp reached top.
+     */
+    tiltUp: boolean;
+    /**
+     * TiltDown reached bottom.
+     */
+    tiltDown: boolean;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate the reason for the error.
+   */
+  export enum ErrorReason {
+    CommandResetError,
+    CommandZoomInError,
+    CommandZoomOutError,
+    CommandPanLeftError,
+    CommandPanRightError,
+    CommandTiltUpError,
+    CommandTiltDownError,
+    SendDataError,
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Data structure to indicate the status of the current session.
+   */
+  export interface SessionStatus {
+    /**
+     * Whether the far-end user is controlling a  device.
+     */
+    inControl: boolean;
+    /**
+     * Id of  participant whose device is being controlled.
+     */
+    id: string;
+    /**
+     * Reason the  control session was terminated.
+     */
+    terminatedReason?: SessionTerminatedReason;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Enum used to indicate the reason the session was terminated.
+   */
+  export enum SessionTerminatedReason {
+    None,
+    ControlDenied,
+    ControlNoResponse,
+    ControlBusy,
+    AckTimeout,
+    ControlTerminated,
+    ControllerTerminated,
+    DataChannelError,
+    ControllerCancelled,
+    ControlDisabled,
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Fetch a list of the participants with controllable-cameras in a meeting.
+   * @param callback Callback contains 2 parameters, error and result.
+   * error can either contain an error of type SdkError, incase of an error, or null when fetch is successful
+   * result can either contain the CapableParticipants object, incase of a successful fetch or null when it fails
+   * result: object that contains an array of participants with controllable-cameras
+   */
+  export function getCapableParticipants(callback: (error: SdkError, result: CapableParticipants) => void): void {
```

Comment:
why not just return Participant[] ? what the use of class that has just one field.